Sources/Converters.swift

```diff
@@ -753,3 +813,101 @@ class DropoutConverter: NodeConverter {
 
     }
 }
+
+@available(iOS 11.0, macOS 10.13.0, tvOS 11.0, *)
+final class AbsConverter: NodeConverter {
+    func convert(in graph: ONNXGraph, node: Onnx_NodeProto) throws {
+        guard
+            let input = graph.output(name: node.input[0]),
+            let inputShape = graph.shape(output: node.input[0])
+        else { throw ONNXGraph.Errors.noSuchOutput }
+
+        let abs = MPSCNNNeuronAbsoluteNode(source: input)
+        abs.label = "Abs"
+        graph.addFilter(abs, outputShape: inputShape, withOutputs: node.output)
+    }
+}
+
+@available(iOS 11.0, macOS 10.13.0, tvOS 11.0, *)
+final class HardSigmoidConverter: NodeConverter {
+    func convert(in graph: ONNXGraph, node: Onnx_NodeProto) throws {
+        guard
+            let input = graph.output(name: node.input[0]),
+            let inputShape = graph.shape(output: node.input[0])
```

Comment:
Is it safe to access the elements by index here without checking `.count` like you previously did?
Maybe should add `guard let nodeInputFirst = node.input.first, ...`?